src/services/contentService.ts

```diff
@@ -9,7 +9,7 @@ export const getHomePageContent = async (): Promise<Page> => {
   if (MOCK_API_RESPONSES !== 'true') {
     console.log('api_path');
 
-    const homePageId = "home-page";
+    const homePageId = "609d4a26a90f85a3c7421921";
```

Comment:
Should this change be included?

I understand that we haven't fully decided the correct way to access pages yet, so keeping "home-page" is wrong too, but the new value is dependent on the machine strapi is running on, right?